src/index.js

```diff
@@ -35,6 +35,16 @@ const EmojiPicker = ({
   pickerStyle = {},
   groupVisibility = {},
   searchPlaceholder = null,
+  customGroups = {
+    name: 'Custom Emojis',
+    icon: 'https://www.svgrepo.com/show/154059/custom-coding.svg',
+    emojis: [
+      {
+        name: 'test',
+        url: 'https://cdn-icons-png.flaticon.com/512/1300/1300455.png',
+      },
+    ],
+  },
```

Comment:
I believe you used this for testing, it should probably not be merged to master :)

Also, just as a note `customGroups` hints it should be an array of groups, so it should either be customGroup, or accept an Array.

```suggestion
  customGroups = {},
```

Reply:
fixed on commit 97e16ba5f2c43bcbbb55d8a425e76960c8581fef